konduit-serving-io/konduit-serving-camera/src/main/java/ai/konduit/serving/camera/step/capture/CameraFrameCaptureStep.java

```diff
@@ -22,33 +22,36 @@
 import io.swagger.v3.oas.annotations.media.Schema;
```

Comment:
konduit-serving-camera - along with the parent module konduit-serving-io - should have been removed entirely.
I see you've copied the content here to konduit-serving-image (which is good) but the old copy has to go too.